.devcontainer/docker-compose.yml

```
@@ -0,0 +1,67 @@
+    container_name: database_dozens
+    restart: unless-stopped
+    networks:
+      - dozens-ecosystem
+    ports:
+      - '5432:5432'
+    environment:
+      - POSTGRES_USER=dozens
+      - POSTGRES_PASSWORD=dozens
+      - POSTGRES_DB=dozens
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U dozens" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)